selfupdate/detect_test.go

```diff
@@ -51,6 +52,32 @@ func TestDetectReleaseWithVersionPrefix(t *testing.T) {
 	}
 }
 
+func TestDetectVersionExisting(t *testing.T) {
+	r, ok, err := DetectVersion("rhysd/github-clone-all", "2.3.0")
```

Comment:
2.3.0 is the current latest. To check if the function can detect the older version, please use 2.2.0 instead.